### PR TITLE
oauth-apiserver: modify dockerfile paths to match other master repos

### PR DESCRIPTION
**Pull request by stlaz**
Ref: https://github.com/openshift/oauth-apiserver/pull/5

### CODE REVIEW

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ build_root:
     namespace: openshift
     tag: golang-1.12
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]: shouldn't this be 1.13 now?

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ build_root:
...
 images:
-- dockerfile_path: Dockerfile.rhel7
+- dockerfile_path: images/Dockerfile.rhel
   from: base
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]: this should carry the builder image replacement section

**Comment by stlaz** [comment]
@tnozicka thx for the review, I had those changes prepared and accidentally pushed them to a different branch on Friday, I think your comments should be addressed now.

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-release-4.5.yaml [hunk 3]
```diff
@@ -8,10 +8,15 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.12
+    tag: golang-1.13
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-release-4.5.yaml]: you don't have to edit these branches - all above master (not cut yet) are fast forwarded - it also limits the CI to branches that you care about - 4.5 is likely flaky as it's being setup just to test the configs

**Comment by tnozicka** [comment]
otherwise it lgtm, I'll tag on green rehearsal, the unit might be connected to golang version bump
```
    roundtrip.go:330: OAuthAuthorizeToken: diff:   &oauth.OAuthAuthorizeToken{
          	TypeMeta: v1.TypeMeta{},
          	ObjectMeta: v1.ObjectMeta{
          		... // 13 identical fields
          		Finalizers:    nil,
          		ClusterName:   "då˜¾Yåƒ¿æ‹»ä¾©é§·^jM,zVå–",
        - 		ManagedFields: []v1.ManagedFieldsEntry{},
        + 		ManagedFields: nil,
          	},
          	ClientName: "",
          	ExpiresIn:  5218071493905793220,
          	... // 4 identical fields
          	UserUID:             "èŒŒÄ­Æ°É¶æŽ¶çŽ´æ—™",
          	CodeChallenge:       "{3æ¤‰ÈŒbæ­§Æ´G",
        - 	CodeChallengeMethod: "",
        + 	CodeChallengeMethod: "plain",
```

**Comment by stlaz** [comment]
I removed all the changes from the 4.5 file.

It kind of looks like some defaulting is actually broken, but that's an issue in the very oauth-apiserver repo itself

**Comment by tnozicka** [comment]
yeah, the repo was not yet working, so CI should go first and the rehearsal be fixed with regular PR with CI passing

/lgtm

**Pull request merged**